main.py

```diff
@@ -36,6 +36,13 @@
     'CAM_BACK_RIGHT': 'ring_side_right',
 }
 
+CITY_TO_ID = {
```

Comment:
can we add a brief inline comment here saying:
```python
# 3-letter abbreviation of nuScene city names
CITY_TO_ID = {
...
```

Reply:
Done